apps/token-manager/app/src/App.js

```diff
@@ -41,31 +42,45 @@ class App extends React.Component {
       })
     }
   }
-  handleAssignTokens = ({ amount, recipient }) => {
+  handleUpdateTokens = ({ mode, amount, holder }) => {
     const { app, tokenDecimalsBase } = this.props
-    const amountConverted = Math.floor(parseFloat(amount) * tokenDecimalsBase)
-    const toMint = new BN(`${amountConverted}`, 10)
-    app.mint(recipient, toMint.toString())
+    const amountRounded = Math.floor(parseFloat(amount) * tokenDecimalsBase)
+    const amountValue = new BN(`${amountRounded}`, 10).toString()
+
+    if (mode === 'assign') {
+      app.mint(holder, amountValue)
+    }
+    if (mode === 'remove') {
+      app.burn(holder, amountValue)
```

Comment:
@sohkai do you know why this doesn’t work? `BURN_ROLE` doesn’t appear in the `artifact.json`.

Reply:
We completely forgot it before on 0.5. The current Rinkeby instances would need a redeployment with a re-gened `artifact.json` now that https://github.com/aragon/aragon-apps/pull/477 is merged.

Let's just make sure this works on the new DAOs :).